tests/itests/src/test/java/org/codice/ddf/admin/query/request/LdapRequestHelper.java

```diff
@@ -69,6 +70,7 @@ public void waitForLdapInSchema() {
   }
 
   public void waitForConfigs(List<Map<String, Object>> expectedConfigs, boolean ignorePid) {
+    Gson gson = new GsonBuilder().create();
```

Comment:
❓ Should we just create this once as you did everywhere else and make it static and final?

Reply:
Good catch missed this one